melodic/distribution.yaml

```diff
@@ -1319,6 +1319,16 @@ repositories:
       url: https://bitbucket.org/DataspeedInc/dataspeed_ulc_ros.git
       version: master
     status: developed
+  dbc:
```

Comment:
> It is not my package but I needed it in rosdistro for the industrial CI to succeed.

@Timple: instead of adding someone else's package to `rosdistro`, you could consider using a `.rosinstall` file that adds this dependency to your CI workspace. See [here](https://github.com/ros-industrial/industrial_ci/blob/legacy/doc/index.rst#build-depended-packages-from-source) for the documentation on setting that up.

Reply:
@gavanderhoorn: Yes, you are right. `rosdep` (and therefore `industrial_ci`) does not support installing the source version.

>  instead of adding someone else's package to rosdistro, you could consider using a .rosinstall file that adds this dependency to your CI workspace

Or the new [inline syntax](https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#workspace-definition) :sunglasses:

Reply:
@Timple If you're looking for it to be resolved by rosdep this will not be enough. It will have to be released to be considered part of a rosdistro and then resolved by rosdep while that rosdistro is active. Unless you want to maintain the package I would suggest that you encourage the upstream maintainers to consider it with a request for release. And in the meantime look at using a rosinstall file as suggested above. And we'll close this.